src/main/java/net/geforcemods/securitycraft/blocks/reinforced/ReinforcedLeverBlock.java

```diff
@@ -0,0 +1,101 @@
+package net.geforcemods.securitycraft.blocks.reinforced;
+
+import net.geforcemods.securitycraft.api.OwnableTileEntity;
+import net.geforcemods.securitycraft.misc.OwnershipEvent;
+import net.geforcemods.securitycraft.tileentity.ReinforcedButtonTileEntity;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.block.LeverBlock;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.particles.RedstoneParticleData;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ActionResultType;
+import net.minecraft.util.Direction;
+import net.minecraft.util.Hand;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvents;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.BlockRayTraceResult;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.World;
+import net.minecraft.world.storage.loot.LootContext;
+import net.minecraftforge.common.MinecraftForge;
+
+import java.util.List;
+
+public class ReinforcedLeverBlock extends LeverBlock implements IReinforcedBlock {
+
+    public ReinforcedLeverBlock(Properties properties)
+    {
+        super(properties);
+    }
+
+    @Override
+    public ActionResultType onBlockActivated(BlockState state, World world, BlockPos pos, PlayerEntity player, Hand hand, BlockRayTraceResult result) {
+        BlockState newState;
+        if (world.isRemote) {
+            newState = state.cycle(POWERED);
+            if (newState.get(POWERED)) {
+                addParticles(newState, world, pos, 1.0F);
+            }
+
+            return ActionResultType.SUCCESS;
+        } else {
+            newState = this.func_226939_d_(state, world, pos);
+            float pitch = (Boolean)newState.get(POWERED) ? 0.6F : 0.5F;
+            world.playSound((PlayerEntity)null, pos, SoundEvents.BLOCK_LEVER_CLICK, SoundCategory.BLOCKS, 0.3F, pitch);
+            return ActionResultType.SUCCESS;
+        }
+    }

```

Comment:
Shouldn't this be a check for the owner/whitelist like for the button? Though in this case because of the client side part of this method, the check needs to be done at the else branch.

Reply:
The way you did it now will not spawn particles if the player is not allowed to press the lever.

Reply:
I don't exactly know what you mean by that. The reinforced lever emits particles just like the normal one, and if a player can't use it, there will be no activation particles (ofc not, as it's state doesn't change), and if the lever is on, the lever on ambient particles are also spawned if I can't interact with the lever.

Reply:
Right, I'm just dumb. I misinterpreted the code.